tests/unit/amo/components/TestReportAbuseButton.js

```diff
@@ -304,4 +308,16 @@ describe(__filename, () => {
     // method was called.
     sinon.assert.called(fakeEvent.preventDefault);
   });
+
+  it('renders an error if one exists', () => {
+    const errorHandler = createStubErrorHandler();
+    const { store } = dispatchClientMetadata();
+    store.dispatch(setError({
+      error: new Error('something bad'),
+      id: errorHandler.id,
+    }));
+    const root = renderMount({ errorHandler, store });
```

Comment:
I am not sure if it is relevant here, but in many other places we did this instead:

https://github.com/mozilla/addons-frontend/blob/5a33c7458ef20c33a27cbed1fb73d0bf2577715a/tests/unit/amo/components/TestAddonReviewList.js#L289-L299

Reply:
I saw that but it seems a bit more verbose way to do it... the error is still being dispatched and clearly showing up in the component in my test path, so I'm not sure what the extra tests gain.

Reply:
I think it's ok as is because it dispatches `setError()` with a real store. 

If I recall correctly, the `AddonReview` test suite couldn't use this approach because its render method wasn't triggering Redux state mapping like it should have. It was rendering a base component directly.